integrations/dcgm/ops_agent_metadata.yaml

```diff
@@ -56,3 +57,120 @@ platforms:
     - gpu_number
     - model
     - uuid
+- type: GCE
+  launch_stage: GA
+  version: '2'
+  install_documentation_url: https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party-nvidia
+  agent_requirement:
+    metrics_minimum_supported_version:
+      major: 2
+      minor: 38
```

Comment:
Should this be 2.51.0 according to the linked docs?

Reply:
Done!